manual/3.1/index.html

```diff
@@ -23,7 +23,8 @@
     <script type="text/javascript" src="_static/jquery.js"></script>
```

Comment:
Committing built documentation in the same repository as the code seems like a bad idea.
The repository is pollute with generated HTML files added to those we actually need to review.
Is it the intended approach?